source/more-info/adr0004.markdown

```diff
@@ -0,0 +1,12 @@
+---
+title: "Webscraping"
+description: "With ADR0004 integrations that preform webscraping are not allowed and will be removed."
+sidebar: false
+---
+
+The integration you requested has been removed because it was doing webscraping.
+This is not allowed. To learn more about that decision, [have a look here][ADR0004].
```

Comment:
```suggestion
The integration you requested has been removed because it was doing web scraping.
This is not allowed. To learn more about that decision, [have a look here][ADR-0004].
```